lib/linux_admin/time_date.rb

```diff
@@ -0,0 +1,21 @@
+module LinuxAdmin
+  class TimeDate
+    extend Common
+    COMMAND = 'timedatectl'
+
+    def self.timezone
+      result = run(cmd(COMMAND), :params => ["status"])
+      result.output.split("\n").each do |l|
+        return l.split(':')[1].strip if l =~ /Time.*zone/
+      end
+    end
+
+    def self.set_system_time(time)
```

Comment:
I think an accessor looking method would be better: `def self.system_time=(time)`

```ruby
LinuxAdmin::TimeDate.system_time = Time.now
```